web/packages/teleterm/src/ui/DocumentAuthorizeWebSession/DocumentAuthorizeWebSession.tsx

```diff
@@ -114,6 +117,30 @@ export function DocumentAuthorizeWebSession(props: {
               This device is not trusted
             </Alert>
           )}
+          {!isRequestedUserLoggedIn && (
+            <Alert
+              mb={0}
+              primaryAction={{
+                content: 'Log Out',
+                onClick: () => {
+                  ctx.commandLauncher.executeCommand('cluster-logout', {
+                    clusterUri: rootCluster.uri,
+                  });
+                },
+              }}
+              details={
+                <>
+                  You are logged in as <b>{rootCluster.loggedInUser?.name}</b>.
+                  To authorize this web session request, please log out and log
+                  in again as <b>{props.doc.webSessionRequest.username}</b>.
+                  <br />
+                  Then retry the action in the browser.
```

Comment:
This is fine, but "Then retry the action in the browser" seems too... generic(?) to me. Is it possible to say the exact button text in the brower like "Then click 'Launch Teleport Connect' again in the browser" ?

but even that seems weird cuz they might think it means to launch Teleport Connect again and not click the button.

Idk, i'm ok with whatever but it seems slightly off.

Reply:
I think `Then click Launch Teleport Connect again in the browser.` is clear enough. Thanks!